README.md

```diff
@@ -126,3 +126,13 @@ It should be stated that the abstraction layer (currently `NAN`) should
 support all *current* LTS releases. Given that Active LTS will overlap
 for a period of 6 months, this means that the abstraction layer will, at
 any given point in time, fully support a maximum of 2 LTS releases.
+
+## LTS Team members
+
```

Comment:
Anyone in the Nodejs org can see team membership. It's not useful for people outside the org, but still helpful.